### PR TITLE
Add icons gallery to storybook

### PR DESCRIPTION
**Pull request by pallendes**
This PR adds an Icons gallery to storybook

image

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,3 +2,4 @@ dist
 node_modules
 storybook-static
 .idea
+.vscode
```
Comment (rodelta) [inline .gitignore:5]: I don't think this belongs in the gitignore. That folder allows for config for the repo, and it's a good practice to have it checked in. 

What setting are you using?

File: src/icons/icons.mdx [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+import * as Icons from ".";
+
+<Meta title="Icons gallery" />
+
+# Icons Gallery
+
+List of available icons
+
+<IconGallery>
```
Comment (rodelta) [inline src/icons/icons.mdx:11]: I think you can also do this with a normal story file, what advantage are we getting for mdx?
Curious about it, I don't have a lot of experience with mdx

File: .gitignore (hunk 1 above)
Comment (pallendes) [reply .gitignore:5]: Ok, I can include it then.

File: src/icons/icons.mdx (hunk 2 above)
Comment (pallendes) [reply src/icons/icons.mdx:11]: The `<IconGallery>` component doesn't work if it's not used inside a mdx file. In addition to that, Icon gallery is not technically a story, I did it like that before this (creating my own `IconGallery` component, and so on), and I didn't like the result.

**Review by rodelta — APPROVED** [review]

File: src/icons/icons.mdx (hunk 2 above)
Comment (rodelta) [reply src/icons/icons.mdx:11]: Makes sense. Thanks for the clarification!

**Pull request merged**